solidity-checks/src/main/java/org/sonarsource/solidity/checks/CheckUtils.java

```diff
@@ -0,0 +1,14 @@
+package org.sonarsource.solidity.checks;
+
+public class CheckUtils {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Add a private constructor to hide the implicit public one. ([squid:S1118](https://sonarcloud.io/organizations/stylianos-agapiou-sonarsource-gi/rules#rule_key=squid%3AS1118))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=org.sonarsource.solidity%3Asonar-solidity&pullRequest=24&issues=AWRgYslXl6vpliCe2G5E&open=AWRgYslXl6vpliCe2G5E)

[ISSUE_KEY]: AWRgYslXl6vpliCe2G5E